src/components/InfoTip/MenuFooter.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class MenuFooter extends React.Component {
+  render() {
+    const { children } = this.props;
```

Comment:
I think it make sense to pass down all the given props to the footer div.

```jsx
const { children, ...props } = this.props;

return <div className="footer" {...props}>{children}</div>;
```

Reply:
Can also add the option to add more classes:
https://github.com/patternfly/patternfly-react/blob/master/src/components/Alert/Alert.js#L19-L21